.travis.yml

```diff
@@ -34,7 +34,7 @@ services:
   - redis-server
 
 before_install:
-  - phpenv config-rm xdebug.ini
+  - if [[ $TRAVIS_PHP_VERSION != 7.2 ]] ; then phpenv config-rm xdebug.ini; fi
```

Comment:
This is brittle with time. We should check if that config exists, rather than a switch on the php version.